internal/version.go

```diff
@@ -17,10 +18,15 @@ var (
 // This is because release-please keeps this at the released version, and not the upcoming next version.
 // While the next version may not match the patch release, it causes the right behavior for semver version comparisons.
 func FullVersion() string {
-	v, _ := semver.NewVersion(Version)
+	v, err := semver.NewVersion(Version)
+	if err != nil {
+		v, _ = semver.NewVersion(realVersion)
+	}
```

Comment:
Instead of real version, what do you think about?


```suggestion
	v, err := semver.NewVersion(Version)
	if err != nil {
		panic(fmt.Sprintf("invalid version %v: %v", Version, err))
	}
```

Still a panic, but should make it easier to debug.